docker-compose.yaml

```diff
@@ -7,9 +7,6 @@ services:
             dockerfile: docker/dockerfiles/web
             args:
                 - POETRY_ARGS=  # allow dev install
-        working_dir: /srv/app
-        command: ./manage.py runserver_plus 0.0.0.0:8000
-        user: ${UID:-1000}  # user id or default
```

Comment:
This is fine.  I'll add it back.  The main thing was getting the uid into the Dockerfile.

Reply:
I think putting back the other two lines is helpful for docker compose development processes as well.

Reply:
Since working dir isn't usually different between dev and prod, keeping it in 2 locations makes maintenance a bit of a headache.  Since the Dockerfile is canonical across multiple deployment options, including docker-compose, keeping it there makes sense.  The Command does need to be put back, since it is different between environments.  I'm still looking into how user is handled, as it does appear that the compose version is different, but I'm not clear on how.